internal/types/types.go

```diff
@@ -144,7 +144,7 @@ func GetDynamicStyle(str string) lipgloss.Style {
 	h.Write([]byte(str))
 	hash := h.Sum32()
 
-	color := colors[int(hash)%len(colors)]
+	color := colors[hash%uint32(len(colors))]
```

Comment:
Please add tests for what you are fixing.

You have examples in the referenced issues, right?